### ISSUE TITLE
UGen for removing bad values

### ISSUE THREAD

**Issue by snappizz** [supercollider/supercollider#2512]
This has been discussed a few times. I would like to see a real UGen that does this so it is accessible to alternate clients.

wslib has the [RemoveBadValues](https://github.com/supercollider-quarks/wslib/blob/master/wslib-classes/Extensions/UGens/RemoveBadValues.sc) pseudo-ugen. For names, I suggest `ReplaceBadValues`, `ZapGremlins`, or `Sanitize`.

**Comment by telephon** [comment]
It could be even a UGen method (like `range`), so you can quickly add it to a UGen at any place in the code.

**Comment by muellmusik** [comment]
As I've said this would be trivial to copy and modify CheckBadValues for this. I'm a bit swamped at the moment, but basically replace the switch statement with something like (untested):

```
if(classification != FP_NORMAL) { ZXP(out) = 0 } else { ZXP(out) = samp };
```

**Comment by snappizz** [comment]
@telephon if it's going to be a UGen method then it should also be a SimpleNumber method, which means we should add some sclang primitive wrapper for `std::fpclassify`

**Comment by telephon** [comment]
https://github.com/supercollider/supercollider/pull/2514

**Comment by snappizz** [comment]
Fixed in #2561.
